## This is the color description for each scene
## Related pkg: visor
##
## Author: Ferdian Pratama

Header header
visor/colorInfo[] object_color
int16 object_count